### ISSUE TITLE
somalier (sample control + ancestry)

### ISSUE THREAD

**Issue by egenomics** [nf-core/raredisease#446]
### Description of feature

Implement the software somalier to have a quality control to detect sample swaps, as well as some output on the ancestry of the sample, very useful to interpret a clinical sample.

https://github.com/brentp/somalier

A possible alternative for ancestry would be rye:
https://github.com/healthdisparities/rye

**Comment by jemten** [comment]
Good idea, we're already using peddy from the same author to do ancestry, relatedness and sex check. He is suggesting switching to somalier as the more scalable solution. Will look into this for upcoming releases
